### PR TITLE
feat(jenkins_master): Add CSRF config for Jenkins Masters.

### PR DESCRIPTION
**Pull request by jtk54**
Relies on https://github.com/spinnaker/igor/pull/222.

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/jenkins/JenkinsAddMasterCommand.java [hunk 1]
```diff
@@ -49,12 +49,19 @@ protected String getCiName() {
...
+  public Boolean csrf;
+
   @Override
   protected Master buildMaster(String masterName) {
     JenkinsMaster master = (JenkinsMaster) new JenkinsMaster().setName(masterName);
     master.setAddress(address)
         .setPassword(password)
-        .setUsername(username);
+        .setUsername(username)
+        .setCsrf(csrf);
```
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/jenkins/JenkinsAddMasterCommand.java]: OK if this is null?

Comment (jtk54) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/jenkins/JenkinsAddMasterCommand.java]: Yes, it defaults to false in Igor. Should I add a default in hal also?

Comment (lwander) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/jenkins/JenkinsAddMasterCommand.java]: No that's fine then, just checking

**Comment by jtk54** [comment]
Docs generated.

**Pull request merged**